code/__defines/_macros.dm

```diff
@@ -33,6 +33,7 @@
 
 #define isAI(A) istype(A, /mob/living/silicon/ai)
 #define isDrone(A) istype(A, /mob/living/silicon/robot/drone)
+#define isStandardDrone(A) (A.type == /mob/living/silicon/robot/drone)
```

Comment:
You should use ispath here, though a solution using a boolean proc on the mob is preferred.